server/database/config/build.sql

```diff
@@ -4,16 +4,18 @@ DROP TABLE IF EXISTS users, exercises, food, food_category, food_type, nutrition
 
 CREATE TABLE users(
 id SERIAL PRIMARY KEY,
-first_name VARCHAR(100) NOT NULL,
-last_name VARCHAR(100) NOT NULL,
-email VARCHAR (100) NOT NULL UNIQUE,
-password VARCHAR(250) NOT NULL ,
-gender CHAR(1),
-height INTEGER,
+username VARCHAR(100),
```

Comment:
here we don't need usename and name , we decide to change it to first name and last name